pagopa/io-app
# chore(IT Wallet): [SIW-524] Update ITW home screen navbar labels

## Short description
This PR updates the navbar labels in the ITW home screen, namely ITW_HOME_SCREEN.

| Before | After |
|--------|--------|
| <img src="https://github.com/pagopa/io-app/assets/12371664/2b39545f-a4e2-4fc7-a4ac-ba7088d538c8" height="400"> | <img src="https://github.com/pagopa/io-app/assets/12371664/1ce2ed6c-328f-460b-92eb-f0b78db30cde" height="400"> |

## List of changes proposed in this pull request
- Updates locales;
- Updates the labels array in the screen.

## How to test
Start the app and open the "Wallet" section, the label should be updated with the new values.
